### PR TITLE
Enable inline-require transform for tests.

### PR DESCRIPTION
**Pull request by cpojer**
Relay diff incoming too.

### CODE REVIEW

**Comment by cpojer** [comment]
See https://github.com/facebook/relay/pull/187

File: scripts/babel/inline-requires.js [hunk 1]
```diff
@@ -0,0 +1,128 @@
+/**
+ * Copyright 2004-present Facebook. All Rights Reserved.
```
Comment (zpao) [inline scripts/babel/inline-requires.js]: fixit

File: scripts/babel/inline-requires.js [hunk 2]
```diff
@@ -0,0 +1,128 @@
...
+ */
+
+'use strict';
+/*jslint node: true */
```
Comment (zpao) [inline scripts/babel/inline-requires.js]: rm

**Comment by zpao** [comment]
:thumbsup:

**Pull request merged**